Lib/ExampleMediaPlatform/ExampleMediaPlatformAuthContainer.php

```diff
@@ -0,0 +1,10 @@
+<?php
+App::uses('AuthContainer', 'AuthManager.Lib');
+
+/**
+ * Class ExampleMediaPlatformAuthContainer
+ */
+class ExampleMediaPlatformAuthContainer extends AuthContainer {
+
```

Comment:
Misschien hier even een comment waarin wordt verteld waar deze klasse voor is. Het is natuurlijk een beetje verwarrend omdat er geen methods in staan.